Solutions/Proofpoint On demand(POD) Email Security/Parsers/ProofpointPOD.yaml

```diff
@@ -314,4 +379,245 @@ FunctionQuery: |
         | project-away Guid1
         | project-rename Urls = set_URLLIST
     };
-    union isfuzzy=true ProofpointPOD_message_view, ProofpointPOD_maillog_view
\ No newline at end of file
+    let ProofpointPOD_messageV2_view = view () { 
+        ProofpointPODMessage_CL
+        | extend 
+                FilterModulesUrldefenseCountsNoRewriteIsLargeMsgPartSize= toreal(parse_json(filter).modules.urldefense.counts.noRewriteIsLargeMsgPartSize),
+    		        PpsVersion= tostring(parse_json(pps).version),
+    		        PpsCid= tostring(parse_json(pps).cid),
+    		        MsgParts=column_ifexists('msgParts', ''),
+    		        MetadataOriginDataAgent= tostring(parse_json(metadata).origin.data.agent),
+    		        MetadataOriginDataVersion= tostring(parse_json(metadata).origin.data.version),
+    		        MetadataOriginDataCid= tostring(parse_json(metadata).origin.data.cid),
+    		        ConnectionHelo= tostring(parse_json(connection).helo),
+    		        MsgLang= tostring(parse_json(msg).lang),
+    		        MsgNormalizedHeaderSubject= tostring(parse_json(msg).normalizedHeader.["subject"]),
```

Comment:
avoid using parse_json for same field multiple time. Same comment as previous one